corehq/blobs/tests/test_fsdb.py

```diff
@@ -116,6 +120,31 @@ def test_put_with_colliding_blob_id(self):
         with self.db.get(key=meta.key) as fh:
             self.assertEqual(fh.read(), b"bang")
 
+    def test_expire(self):
+        now = datetime.utcnow()
+        meta = self.db.put(
+            BytesIO(b"content"),
+            domain="test",
+            parent_id="test",
+            type_code=CODES.tempfile,
+        )
+        with self.db.get(key=meta.key) as fh:
+            self.assertEqual(fh.read(), b"content")
+        self.db.expire("test", meta.key)
+
+        future_date = now + timedelta(minutes=120)
+        with temporary_blob_db(self.db), \
+                patch('corehq.blobs.tasks._utcnow', return_value=future_date):
+            delete_expired_blobs()
+
+        with self.assertRaises(mod.NotFound):
+            self.db.get(key=meta.key)
+
+    def test_expire_missing_blob(self):
+        self.db.expire("test", "abc")  # should not raise error
+        with self.assertRaises(mod.NotFound):
```

Comment:
nit: not sure if this is worth testing. Its ensuring that `get` raises an error which is related to `get` and should not concern `expire` unless you are trying to cover what's happening within `expire`. I'd still vote against it.